libs/src/sdk/config/types.ts

```diff
@@ -1,4 +1,7 @@
 export type BootstrapConfig = {
```

Comment:
Makes me wonder if we should rename this to just `Config` or `SdkConfig`

Reply:
yeah I agree

Reply:
how about ServicesConfig?

Reply:
renamed!